packages/gatsby/src/query/query-runner.ts

```diff
@@ -172,10 +172,7 @@ export async function queryRunner(
   }
 
   const resultJSON = JSON.stringify(result)
-  const resultHash = crypto
-    .createHash(`sha1`)
-    .update(resultJSON)
-    .digest(`base64`)
+  const resultHash = await sha1(resultJSON)
```

Comment:
This _technically_ changes behavior since `sha1` returns `hex`, not `base64`

Reply:
That's fine - we don't care about exact values here, only that we can detect result changes in "performant" way